scenarios/test_codegen/test/lib_tests/Table_test.res

```diff
@@ -1,158 +0,0 @@
-open Table
-open RescriptMocha
-
-let isPrimaryKey = true
-
-describe("Table functions postgres interop", () => {
-  it("Makes batch set function for entity", () => {
-    let table = mkTable(
-      "test_table",
-      ~schemaName="public",
-      ~fields=[
-        mkField("id", Text, ~fieldSchema=S.string, ~isPrimaryKey),
-        mkField("field_a", Numeric, ~fieldSchema=S.bigint),
-      ],
-    )
-
-    let batchSetFnString = table->PostgresInterop.makeBatchSetFnString
-
-    let expected = `(sql, rows) => {
-      return sql\`
-        INSERT INTO "public"."test_table"
-        \${sql(rows, "id", "field_a")}
-        ON CONFLICT(id) DO UPDATE
-        SET
-        "id" = EXCLUDED."id", "field_a" = EXCLUDED."field_a";\`
-    }`
-
-    Assert.equal(batchSetFnString, expected)
-  })
-
-  it("Makes batch set function for entity with custom schema", () => {
-    let table = mkTable(
-      "test_table",
-      ~schemaName="custom",
-      ~fields=[
-        mkField("id", Text, ~fieldSchema=S.string, ~isPrimaryKey),
-        mkField("field_a", ~fieldSchema=S.bigint, Numeric),
-      ],
-    )
-
-    let batchSetFnString = table->PostgresInterop.makeBatchSetFnString
-
-    let expected = `(sql, rows) => {
-      return sql\`
-        INSERT INTO "custom"."test_table"
-        \${sql(rows, "id", "field_a")}
-        ON CONFLICT(id) DO UPDATE
-        SET
-        "id" = EXCLUDED."id", "field_a" = EXCLUDED."field_a";\`
-    }`
```

Comment:
Do these kinds of tests still exist somewhere else? I think I remember you made a new test suite for it.

Reply:
Yes, there are new tests. Also, it's weird to test code, which is not used :)